core/src/com/unciv/logic/map/mapunit/MapUnit.kt

```diff
@@ -876,6 +876,12 @@ class MapUnit : IsPartOfGameInfoSerialization {
     }
 
 
+    private fun isAlly(otherCiv: Civilization?): Boolean {
```

Comment:
When would this get null as an input? Sounds to be like it's better off not nullable, OR checking that explicitly first

Reply:
Good catch, that was from an earlier commit before I ripped out a bunch of code and forgot to change it back to non-null :D